timiki/rpc-server-bundle/4.0/config/packages/rpc_server.yaml

```diff
@@ -0,0 +1,33 @@
+# Read the documentation: https://github.com/timiki/rpc-server-bundle
+
+rpc_server:
+
+    # mapping configs
+    #
+    # Default mapping
+    #   mapping: '%kernel.root_dir%/Method'
+    #
+    # Multi dir mapping
+    #   mapping:
+    #   - '%kernel.root_dir%/Method1'
+    #   - '%kernel.root_dir%/Method2'
+    #
+    # Multi handler|dir mapping
```

Comment:
This long comment is not needed, right? It should be in the bundles docs.

Reply:
@Nyholm you right about that. Bundle doc contains info about it ) i fix it